avocado/utils/pci.py

```
@@ -376,6 +376,96 @@ def get_vendor_id(full_pci_address):
+
+    :param full_pci_address: Full PCI device address (e.g., 0000:03:00.0)
+    :param driver: Driver to be attached to the specified PCI device
+    :raises ValueError: If `driver` or `full_pci_address` is None, or if the driver
+        could not be attached successfully due to an OS error.
+    :warning: This function may unbind the device from its current driver, which can
+        temporarily make the device unavailable until it is reattached.
+    """
+
+    try:
+        if not driver or not full_pci_address:
+            raise ValueError("'driver' or 'full_pci_address' inputs are None")
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)